node/hello/cbor_gen.go

```diff
@@ -105,7 +105,6 @@ func (t *HelloMessage) UnmarshalCBOR(r io.Reader) error {
 	if maj != cbg.MajUnsignedInt {
 		return fmt.Errorf("wrong type for uint64 field")
 	}
-	t.HeaviestTipSetHeight = uint64(extra)
```

Comment:
This generated code is weird.

Reply:
It won't write to `HeaviestTipSetHeight` ever.

Reply:
May need to regen, I've probably removed it when fixing compilation after merge